get_new_and_deleted.sh

```diff
@@ -1,45 +1,41 @@
-D=$(date +\%Y-\%m-\%d)
-
 # BOIA: 271744772
-comm -2 -3 data/boia_ht_item_ids.$D.txt data/boia_current_ht_item_ids.$D.txt\
-  > data/boia_new_ht_item_ids.$D.txt
-comm -1 -3 data/boia_ht_item_ids.$D.txt data/boia_current_ht_item_ids.$D.txt\
-  > data/boia_deleted_ht_item_ids.$D.txt
+comm -2 -3 data/boia_ht_item_ids.txt data/boia_current_ht_item_ids.txt\
```

Comment:
This file looks like 7 calls to the same function to me, if you want to take it that far.

Reply:
Yup, folding this in to another one of the higher level scripts and DRYing it out into a loop.